src/styles/invite.scss

```diff
@@ -0,0 +1,15 @@
+.invite__form {
```

Comment:
here you can select different layouts by commenting out properties! 
[ ] play around in DevTools
I selected the one closest to the invite screen during signup (input lengths vary with internationalization)